Replicator/ConnectedClient/ConnectedClient.cc

```diff
@@ -90,8 +100,10 @@ namespace litecore::client {
     void ConnectedClient::onHTTPResponse(int status, const websocket::Headers &headers) {
         asCurrentActor([=] {
             logVerbose("Got HTTP response from server, status %d", status);
+            LOCK(_mutex);
```

Comment:
The scope of the lock shouldn't extend past the delegate call; wrap curly braces around these 3 lines.